LunrCoreLmdb/KeyBuilder.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Text;
+
+namespace LunrCoreLmdb
+{
+    internal static class KeyBuilder
+    {
+        #region Fields
+
+        private const string FieldPrefix = "F:";
+
+        public static byte[] BuildAllFieldsKey() => Encoding.UTF8.GetBytes(FieldPrefix);
```

Comment:
No need to do that every time since it only depends on a constant. You can make that a static readonly field that you initialize only once to that value.

Reply:
Good points all. I've converted this to concatenating pre-encoded buffers as suggested.